entrypoint.sh

```diff
@@ -68,17 +68,22 @@ then
   # setup port forwarding
   FORWARDING_PORTS="$(echo ${PORTS:-'0:65535'} | sed 's/[ ,;][ ,;]*/ /g')"
   echo "Forwarding ports: ${FORWARDING_PORTS// /,}"
+  HOST_PORTS="$(echo ${HOST_PORTS:-$FORWARDING_PORTS} | sed 's/[ ,;][ ,;]*/ /g' | sed 's/:/-/g')"
+  echo "Forwarding to host ports: ${HOST_PORTS// /,}"
+
   iptables -t nat -I POSTROUTING -j MASQUERADE
   for forwarding_port in ${FORWARDING_PORTS}
   do
+    host_port=`echo "$HOST_PORTS" | cut -d ' ' -f 1`
+    HOST_PORTS=`echo "$HOST_PORTS" | sed 's/[^ ]* *\(.*\)$/\1/'`
     iptables --table nat --insert PREROUTING \
       --protocol tcp \
       --dport "$forwarding_port" \
-      --jump DNAT --to-destination "$docker_host_ip"
+      --jump DNAT --to-destination "$docker_host_ip:$host_port"
```

Comment:
for me it does not work for port ranges `8000:9000` even `8000-9000` will not work and ends in an error message ` Port `8000-9000' not valid`

Reply:
it does not work for the default port range of `0:65535`

Reply:
however it works for `1:65535`.  maybe i should change the default.  should port `0` be a valid value? WDYT?